doc/release/v3_3_0.md

```diff
@@ -166,6 +166,8 @@ New Features
 #### `yarpdataplayer`
 
 * `info.log` files containing timestamp info are now handled.
+* Added getSliderPercentage method to the rpc port of yarpdataplayer.
```

Comment:
This file is the changelog of the past release, please add the release note like https://github.com/robotology/yarp/pull/2147/commits/87512cba94c66d5b866734c3a25b5271585065e8

Reply:
Should I add it in a new `yarp_3_4` folder, since this is a feature and not a bug fix? Or should I just add another file called `featureYarpdataplayerRpc.md` similar to `fixYarpdataplayerPlayRpc.md` in the `yarp3_3` folder?

Reply:
I'd say `yarp_3_4`, is it right @drdanz ?

Reply:
Done in https://github.com/robotology/yarp/pull/2148/commits/4fbfea7d2b392163c2b17d920a3504028b02a23d

Reply:
Sorry if I see this too late, the folder is the same as the target branch, therefore `master`.
Do you mind changing it again?

Reply:
I have made the changes in https://github.com/robotology/yarp/commit/7b0eca3b378cb8e495fbeafd98bd50f3c267900d

If you could confirm, it's ok? Thank you :)

Reply:
Friendly ping @drdanz